src/amchart/CommonFunnel.js

```diff
@@ -141,6 +147,20 @@
         domNode.style.height = this.size().height + 'px';
 
         this._palette = this._palette.switch(this.paletteID());
+
+        switch (this.labelPosition()) {
+            case "left":
+                this._chart.marginLeft += parseInt(domNode.getBoundingClientRect().width/8);
```

Comment:
Ideally each widget should draw to the max extent available and then it will be the responsibility of the container (the surface) to allocate padding consistently for all widgets?